models/pp_pop_psc_delta.h

```diff
@@ -387,12 +386,6 @@ pp_pop_psc_delta::set_status( const DictionaryDatum& d )
   State_ stmp = S_;      // temporary copy in case of errors
   stmp.set( d, ptmp );   // throws if BadProperty
 
-  // We now know that (ptmp, stmp) are consistent. We do not
-  // write them back to (P_, S_) before we are also sure that
-  // the properties to be set in the parent class are internally
-  // consistent.
-  Archiving_Node::set_status( d );
```

Comment:
archiving node- get and set_status calls seems to be removed only from this model